vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -1,1049 +1,1151 @@
-                        th:href="@{'/staff/module/'+${module.id}+'/sequence/'+${choice.sequence.id}}">[[${choice.sequence.name}]]</a>
-                    <input type="hidden" class="col-md-8" id="deleteChoiceBlockId" th:value="${contents.id}"> 
-                    <a class="btn deleteChoiceBlock col-md-1" href="javascript:;" style="position:absolute; right:0; top: 0;">
+            </div>
+            <div id="slideSpace">
+                <div th:each="contents: ${slideContents}">
+                    <div th:id="${contents.id}" class="valueDiv tab row" th:if="${#strings.equals(contents['class'].simpleName,'ImageBlock')}">
+                        <img th:id="${contents.id}" class="imgDiv" th:src="@{'/api/image/'+${contents.image.id}}" />
+                        <input type="hidden" id="deleteImageId" th:value="${contents.id}"> 
+                        <a class="btn deleteImage col-md-1" href="javascript:;" style="position:absolute; right:25px;"> 
                         <span class="icon-trash error"></span>
-                    </a>
+                        </a>
+                    </div>
+                    <div style="word-break:break-word;" th:id="${contents.id}" class="textDivStaff tab row" th:if="${#strings.equals(contents['class'].simpleName,'TextBlock')}">
+                        <p class="col-md-10"> [[${contents.text}]]</p>
+                        <input type="hidden" id="deleteTextId" th:value="${contents.id}" />
+                        <a class="btn deleteText col-md-1" href="javascript:;"> 
+                        <span class="icon-trash error"></span>
+                        </a>
+                    </div>
+                    <div th:id="${contents.id}" class="choiceDiv tab row" th:if="${#strings.equals(contents['class'].simpleName,'ChoiceBlock')}" style="display:block;">
+                        <a th:if="${contents.showsAll}" th:each="choice: ${choices}" style="display:block; color: var(--primary);"
+                            th:href="@{'/staff/module/'+${module.id}+'/sequence/'+${choice.sequence.id}}">[[${choice.sequence.name}]]</a>
+                        <a th:if="!${contents.showsAll}" th:each="choice: ${contents.choices}" style="display:block; color: var(--primary);"
+                            th:href="@{'/staff/module/'+${module.id}+'/sequence/'+${choice.sequence.id}}">[[${choice.sequence.name}]]</a>
+                        <input type="hidden" class="col-md-8" id="deleteChoiceBlockId" th:value="${contents.id}"> 
+                        <a class="btn deleteChoiceBlock col-md-1" href="javascript:;" style="position:absolute; right:0; top: 0;">
+                        <span class="icon-trash error"></span>
+                        </a>
+                    </div>
                 </div>
             </div>
         </div>
-    </div>
-</body>
-</html>
+    </body>
+</html>
```

Comment:
please avoid reformatting whole files in the future. I can't tell what you've actually changed and what just reformatted.